javaparser-testing/src/test/java/com/github/javaparser/printer/JsonPrinterTest.java

```diff
@@ -9,6 +9,12 @@
 import static com.github.javaparser.utils.Utils.EOL;
 import static org.junit.Assert.*;
 
+import java.io.FileOutputStream;
```

Comment:
I was using this imports for writing the results to file to more easily check that it was valid JSON. You can probably remove these `java.io` imports.